cmd/seq-coordinator-manager/seq-coordinator-manager.go

```diff
@@ -160,11 +164,11 @@ func main() {
 	flex.SetDirection(tview.FlexRow).
 		AddItem(priorityHeading, 0, 1, false).
 		AddItem(tview.NewFlex().
-			AddItem(prioritySeqList, 0, 2, true).
+			AddItem(prioritySeqList, seqManager.maxURLSize+20, 0, true).
```

Comment:
Super nit. 
It's not clear from the code why it should be 20, can you please replace it with a constant so that its clear.

Reply:
Ahh yes, its (9 characters for <- chosen) + (7 characters for dots and bullets) + (3 characters for ellipsis in case URLs are too big) so I set the size to be max+20. 
I didn't think it warranted creating separate constants for this so just added the constant directly. How do you feel about a comment instead of constants?

Reply:
Yeah either is fine.